project/graphql/sssom_schema.graphql

```diff
@@ -7,13 +8,13 @@ type ExtensionDefinition
 
 type Mapping
   {
-    subjectId: EntityReference!
+    subjectId: EntityReference
```

Comment:
What did the `!` mean?

Reply:
Required? I am assuming!

Reply:
It meant the field was _non-nullable_ (see [GraphQL’s doc](https://graphql.org/learn/schema/#object-types-and-fields), which was indeed before #384 the case since the model defined that field as `required`.

Reply:
Thanks!